app/controllers/api/allocations_filtered_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Api
+  class AllocationsFilteredController < AllocationsController
+  private
+
+    PERMITTED_CREATE_FILTER_PARAMS = [
```

Comment:
"create" feels a bit mis-named, as nothing is actually being created...  How about `PERMITTED_FILTER_PARAMS` ?